src/test/java/com/wcc/platform/factories/SetupMentorshipFactories.java

```diff
@@ -176,6 +177,27 @@ public static MentorshipStudyGroupsPage createMentorshipStudyGroupPageTest() {
         createCustomStyleTest());
   }
 
+  /** Test factory for Ad Hoc Timeline Page. */
+  public static MentorshipAdHocTimelinePage createMentorshipAdHocTimelinePageTest(
```

Comment:
Do we really need this method? because looks like it is the same as the default one below:  `var fileName = AD_HOC_TIMELINE.getFileName();`